modules/features/settings/src/main/java/au/com/shiftyjelly/pocketcasts/settings/AutoDownloadSettingsFragment.kt

```diff
@@ -236,18 +238,28 @@ class AutoDownloadSettingsFragment :
         updateView()
     }
 
-    private fun handleNewEpisodesToggle(isEnabled: Boolean) {
+    private fun handleNewEpisodesToggle(status: Int) {
         lifecycleScope.launch {
-            updateNewEpisodesSwitch(isEnabled)
+            if (status == Podcast.AUTO_DOWNLOAD_OFF) {
```

Comment:
We want to disable all auto download podcast status when the user manually disabled the New Episodes toggle